web/assets/main.scss

```diff
@@ -8,4 +8,12 @@ $moj-assets-path: "/supervision/deputies/professional/assets/";
 
 .moj-header__container {
   max-width: 98%;
-}
\ No newline at end of file
+}
+
+.moj-timeline__item {
```

Comment:
Is there a reason these have been put in main.scss rather than pro-hub.scss? The main file should only really be used for imports.

Reply:
no reason - just copied over that way but have moved to the pro-hub.scss file.